### ISSUE TITLE
Missing Model: Seaking 119

### ISSUE THREAD

**Issue by wannefab** [PoGo-Devs/PoGo-3D-Assets#1]

**Comment by Avid29** [comment]
![image](https://cloud.githubusercontent.com/assets/9384045/17831030/99ad3d1c-66ac-11e6-88f6-9e3dafa4bb8c.png)
The model's done but I can't make textures

**Comment by wannefab** [comment]
@Firestarthirty Looks nice so far! Can you work a little bit more on the tail? It doesn't look "fishy" enough, it should have like ripples in it like a cloth you put into water... Also you hav some "bad" polygons in the back, and the rigging need some more tweaking ^^

Thats how it looks in the official app:
https://i.ytimg.com/vi/4EZhspfQr5o/maxresdefault.jpg
https://www.youtube.com/watch?v=WHZq6PedZQM

Btw: Pretty impressive how fast you were! =)

**Comment by Avid29** [comment]
@Lithagon Thank you. I start from a goldeen and venomoth so that made speed pretty easy, I will take your advice to heart and despite the vagueness of "fishy" I know what you mean I didn't remove the features that made it look like a bug (type).

**Comment by Avid29** [comment]
Update! 
-Tail is less like venomoth's wings.
-"Mohawk" is closer to the original design
-Flippers moved forward (closer to original design)
-Horn is rotated, sized and placed better.
![image](https://cloud.githubusercontent.com/assets/9384045/17831605/532e85bc-66bb-11e6-8f8d-e906f06012a2.png)

**Comment by Avid29** [comment]
And rigged uploading now

**Comment by wannefab** [comment]
Nice!
Btw: I found out that the guys from models-ressurce seem to continue ripping the pokemon xy models! I contacted them, maybe they want to support us ☺

**Comment by Avid29** [comment]
@Lithagon Yeah work for nothing!, but I'm sure their model is much better than mine so getting it would be great

**Comment by wannefab** [comment]
I was actually thinking about the unrigged models from the other site ^^ Would give us time to focuse on awesome animations ^^

**Comment by Firestarthirty** [comment]
Ok, quick thing. Is the current master on https://github.com/ST-Apps/PoGo-UWP/tree/master missing references?
